web/src/components/apps/AppVersionHistory.tsx

```diff
@@ -492,89 +492,6 @@ class AppVersionHistory extends Component<Props, State> {
     });
   };
 
-  getVersionDiffSummary = (version: Version) => {
```

Comment:
before i try to refactor `AppVersionHistory` I'm going to try to extract as many parts of it as I can.